scripts/tasks/lint-imports.js

```diff
@@ -138,10 +194,17 @@ module.exports = function() {
     }
 
     if (!fullImportPath || fs.statSync(fullImportPath).isDirectory()) {
-      const pathInvalid = importErrors.pathInvalid;
+      const pathNotFile = importErrors.pathNotFile;
+      const relativePath = path.relative(sourcePath, filePath);
+      pathNotFile.count++;
+      pathNotFile.matches[relativePath] = importPath;
+    }
+
+    if (!allowRelativeImports && pathIsRelative && importPath.indexOf('.scss') === -1) {
```

Comment:
I'm surprised by the exclusion of .scss here. It suggests that .scss file isn't the main "ramification".

Reply:
It is not. 😄